src/lib/utilities.ts

```diff
@@ -366,6 +366,53 @@ function checkProjectValidity(): { isValid: boolean, projectRoot: string } {
   return results;
 }
 
+function createChangelogReadme(
+  versionName: string,
+  changelogPath: string,
+  changeLogData: ChangeLog,
+): void {
+  const createdChangeLogResources = changeLogData[ChangelogConfigTypes.CREATE]?.resources ?? [];
+  const deletedChangeLogResources = changeLogData[ChangelogConfigTypes.DELETE]?.resources ?? [];
+  const updatedChangeLogResources = changeLogData[ChangelogConfigTypes.UPDATE]?.resources ?? [];
+
+  const projectRoot: string | null = getProjectRoot();
+  if (!projectRoot) {
+    return;
+  }
+
+  const createdFiles: string[] = createdChangeLogResources.map(resource => {
+    if (resource.srcPath) {
+      return `${resource.srcPath}/${resource.file?.target}`;
+    }
+
+    return `${resource.file?.target}`;
+  });
+
+  const deletedFiles: string[] = deletedChangeLogResources.map(resource => {
+    if (resource.destPath) {
+      return `${resource.destPath}/${resource.name}`;
+    }
+
+    return `${resource.name}`;
+  });
+
+  const updatedFiles: string[] = updatedChangeLogResources.map(resource => (resource.destPath));
+
+  const readmeContent = `# Changelog - ${versionName}
+The ${'`upgrade`'} command is used to upgrarde a project to the latest version of the template, or to a speciied version.
```

Comment:
not sure whats happening here `${'`upgrade`'} ` seems like its just going to be the string `upgrade`

Reply:
Yea that's correct. it was just a way of escaping the backticks. At the time I forgot about using a backslash. I'll update it